judge/models/problem.py

```diff
@@ -23,6 +24,12 @@
            'License', 'Solution', 'TranslatedProblemQuerySet', 'TranslatedProblemForeignKeyQuerySet']
 
 
+def DisallowCharactersValidator(text):
+    for c in text:
+        if c in settings.DMOJ_PROBLEM_STATEMENT_DISALLOW_CHARACTERS:
+            raise ValidationError(_('Invalid character "%(value)s"'), params={'value': c})
```

Comment:
Potential concern: those who are most likely to make this error are the same ones that cannot distinguish this from the standard quote/double quote.

Reply:
I think rather than "Invalid", we should say "Disallowed" here, since that's more semantically correct.

Reply:
> I think rather than "Invalid", we should say "Disallowed" here, since that's more semantically correct.

fixed